Getting Started with Splitter (#6266)
DevExpress/devextreme-documentation: concepts/05 UI Components/Splitter/00 Getting Started with Splitter/00 Getting Started with Splitter.md

#include tutorials-intro-installationnote

Splitter is a UI component designed to split a page layout into multiple panes.

> This tutorial shows how to add Splitter to the page and configure the component's core settings. As a result, you will create the following page layout:

Reviewer: ```suggestion
This tutorial shows how to add Splitter to the page and configure the component's core settings. For example, you can create the following page layout:
```

> This tutorial shows how to add Splitter to the page and configure the component's core settings. As a result, you will create the following UI component:

Reviewer 2: You say that "Splitter is a UI component" and then you add it to a page and configure. And the result is "the following UI component". So it seems like you used a component to create a component. And I guess that can be true sometimes, but in most cases, you just add it to a page to create a layout that you need. 

```suggestion
This tutorial shows how to add Splitter to the page and configure the component's core settings. As a result, you will create the following page layout:
```

<div class="simulator-desktop-container" data-view="/Content/Applications/24_1/GettingStartedWith/Splitter/index.html, /Content/Applications/24_1/GettingStartedWith/Splitter/index.js, /Content/Applications/24_1/GettingStartedWith/Splitter/index.css"></div>

> Each section in this tutorial describes a single configuration step. You can also find the full source code in the GitHub repository.

Reviewer 2: ```suggestion
Each section in this tutorial describes a single configuration step. You can also find the full source code in the following GitHub repository.
```

> Each section in this tutorial describes a single configuration step. You can also find the full source code in the following GitHub repository.

Reviewer: ```suggestion
Each section in this tutorial describes a single configuration step. You can also find the full source code in the following GitHub repository:
```
